docs/platforms/java/common/configuration/options.mdx

```diff
@@ -145,47 +310,102 @@ This parameter controls whether integrations should capture HTTP request bodies.
+options.maxRequestBodySize = RequestSize.MEDIUM
+```
+```properties {tabTitle:sentry.properties}
+max-request-body-size=MEDIUM
+```
+```bash {tabTitle:Environment Variable}
+SENTRY_MAX_REQUEST_BODY_SIZE="MEDIUM"
+```
+
 </SdkOption>
 
-<SdkOption name="enabled" type="boolean" defaultValue="true">
+<SdkOption name="enabled" type="boolean" defaultValue="true" envVar="SENTRY_ENABLED">
 
 Specifies whether this SDK should send events to Sentry. Setting this to `false` doesn't prevent all overhead from Sentry instrumentation. To disable Sentry completely, depending on environment, call `Sentry.init` conditionally.
 
+```java
+options.setEnabled(false);
+```
+```kotlin
+options.isEnabled = false
+```
+```properties {tabTitle:sentry.properties}
+enabled=false
+```
+```bash {tabTitle:Environment Variable}
+SENTRY_ENABLED=false
+```
+
 </SdkOption>
 
-<SdkOption name="sendClientReports" type="boolean" defaultValue="true">
+<SdkOption name="sendClientReports" type="boolean" defaultValue="true" envVar="SENTRY_SEND_CLIENT_REPORTS">
 
 Set this boolean to `false` to disable sending of client reports. Client reports are a protocol feature that let clients send status reports about themselves to Sentry. They are currently mainly used to emit outcomes for events that were never sent.
 
 _(New in version 6.0.0)_
 
+```java
+options.setSendClientReports(false);
+```
+```kotlin
+options.isSendClientReports = false
+```
+```properties {tabTitle:sentry.properties}
+send-client-reports=false
+```
+```bash {tabTitle:Environment Variable}
+SENTRY_SEND_CLIENT_REPORTS=false
+```
+
 </SdkOption>
 
-<SdkOption name="forceInit" type="boolean" defaultValue="false">
+<SdkOption name="forceInit" type="boolean" defaultValue="false" envVar="SENTRY_FORCE_INIT">
 
 Set this boolean to `true` to force a call to `Sentry.init` to re-initialize the SDK, even if the SDK has already been initialized by a high priority integration.
 
 _(New in version 8.0.0)_
 
+```java
+options.setForceInit(true);
+```
+```kotlin
+options.isForceInit = true
+```
+```properties {tabTitle:sentry.properties}
+force-init=true
+```
+```bash {tabTitle:Environment Variable}
+SENTRY_FORCE_INIT=true
+```
+
 </SdkOption>
 
-<SdkOption name="enableBackpressureHandling" type="boolean" defaultValue="true">
+<SdkOption name="enableBackpressureHandling" type="boolean" defaultValue="true" envVar="SENTRY_ENABLE_BACKPRESSURE_HANDLING">
 
 Set this boolean to `false` to disable automatic downsampling of transactions while the system is under load. The `tracesSampleRate` is halved for every failing health check up to 10 times, (roughly 0.001% of the original `tracesSampleRate`). Any positive health check will reset `tracesSampleRate` to its original value. Health checks run in the background every 10 seconds, checking for queue drops and rate limiting.
 
 Note: Starting with version 7.8.0 backpressure handling has changed from opt-in to opt-out.
 
 _(New in version 7.1.0)_
 
-</SdkOption>
-
-## Integration Configuration
```

Comment:
This seems to have been copied from generic docs and doesn't work this way in Java SDK so I removed it.